ngi_visualizations/fpkm_scatter/fpkm_scatter.py

```diff
@@ -95,12 +131,18 @@ def load_fpkm_counts (file):
             except ValueError:
                 FPKM_idx = 9
             # Iterate through the file
-            for line in fh:
-                line = line.strip()
-                cols = line.split("\t")
-                gene_id = cols[0]
-                FPKM = cols[FPKM_idx]
-                counts[gene_id] = FPKM
+            try:
+                for line in fh:
+                    line = line.strip()
+                    cols = line.split("\t")
+                    gene_id = cols[0]
+                    FPKM = cols[FPKM_idx]
+                    counts[gene_id] = FPKM
+            except IndexError:
+                logging.warn("""One of the input files is not of valid format, are you sure that it's a FPKM file?
```

Comment:
You also have `logging.error` and `logging.critical` levels for stuff like this..